### ISSUE TITLE
Is the number of Labeled data wrong?

### ISSUE THREAD

**Issue by Sleepychord** [eli5168/improved_gan_pytorch#2]
I think you're using 100 labeled data every batch(aka. 1/2 of all data) in your code. Did I misunderstand your code?

**Comment by eli5168** [comment]
Yes, I am using 100 labeled data every batch. 

And you are free to change this number.

**Comment by Sleepychord** [comment]
But this is an semi-supervised learning! We should use a small number of labeled data. Can you reproduce the results in the paper with 1000 labeled data for example?

**Comment by eli5168** [comment]
In this example code, I get 69.7% with 1000 labeled data.

**Comment by Sleepychord** [comment]
emm... You mean that the result in picture was got by setting all 100 in 

```       
(Line 227) label_input = inputv[:100]
           unlabel_input = inputv[100:]
           l_label = real_label[:100]
```
to 4, don't you?

If true, it's okay and thank you.

**Comment by eli5168** [comment]
Yep, I just simply replace this figure with four, though it is not very precise.
